packages/swirl-components/src/components/swirl-tag/swirl-tag.css

```diff
@@ -29,62 +29,146 @@
   padding: var(--s-space-2) var(--s-space-8);
 }
 
-.tag--bordered {
+.tag--variant-outline {
   border-style: solid;
   border-width: var(--s-border-width-default);
   border-color: var(--s-border-strong);
 }
 
+.tag--variant-strong {
+  color: var(--s-text-on-action-primary);
+  background-color: var(--s-surface-neutral-default);
+
+  & .tag__removal-button {
+    color: var(--s-icon-on-action-primary);
+  }
+}
+
 .tag--intent-info {
   color: var(--s-text-info);
   background-color: var(--s-surface-info-subdued);
 
   & .tag__removal-button {
-    color: var(--icon-info);
+    color: var(--s-icon-info);
   }
 
-  &.tag--bordered {
+  &.tag--variant-outline {
     border-color: var(--s-border-info);
   }
+
+  &.tag--variant-strong {
+    color: var(--s-text-on-action-primary);
+    background-color: var(--s-surface-info-default);
+
+    & .tag__removal-button {
+      color: var(--s-icon-on-action-primary);
+    }
+  }
 }
 
 .tag--intent-critical {
   color: var(--s-text-critical);
   background-color: var(--s-surface-critical-subdued);
 
   & .tag__removal-button {
-    color: var(--icon-critical);
+    color: var(--s-icon-critical);
   }
 
-  &.tag--bordered {
+  &.tag--variant-outline {
     border-color: var(--s-border-critical);
   }
+
+  &.tag--variant-strong {
+    color: var(--s-text-on-action-primary);
+    background-color: var(--s-surface-critical-default);
+
+    & .tag__removal-button {
+      color: var(--s-icon-on-action-primary);
+    }
+  }
 }
 
 .tag--intent-warning {
   color: var(--s-text-warning);
   background-color: var(--s-surface-warning-subdued);
 
   & .tag__removal-button {
-    color: var(--icon-warning);
+    color: var(--s-icon-warning);
   }
 
-  &.tag--bordered {
+  &.tag--variant-outline {
     border-color: var(--s-border-warning);
   }
+
+  &.tag--variant-strong {
+    color: var(--s-text-on-action-primary);
```

Comment:
Issue: Contrast is too low. I think we need dark text here.
<img width="1221" alt="Bildschirmfoto 2024-07-17 um 11 33 18" src="https://github.com/user-attachments/assets/1b922711-ef99-4742-a3fa-b038c52637b0">

Reply:
Good catch, thanks.
Talked with the designers and they also agreed.